src/generic_ui/polymer/cloud-install.html

```diff
@@ -58,13 +63,39 @@
     <core-signals on-core-signal-open-cloud-install-dialog='{{ openLoginDialog }}'></core-signals>
 
     <!-- TODO: use core-pages rather than multiple dialogs -->
+    <uproxy-dialog layered='false' backdrop='true' id='regionDialog'>
```

Comment:
Yeah, I added the region picker first because of the one `start` method that @dborkan mentioned. I like the idea of one dialog so I'll go ahead and add the dropdown to `loginDialog` until we have a better idea of the UX flow.